### ISSUE TITLE
Potential circular logic in ag make test

### ISSUE THREAD

**Issue by mortonjt** [biocore/american-gut-web#388]
In this [line](https://github.com/biocore/american-gut-web/blob/master/scripts/ag#L280), I've noticed that psycopg2 will attempt to connect to a database that matches the username (e.g. if the user is ag, it will attempted to connect to the database named ag) and throw an error, since the database doesn't exist

If I create a database named ag, it will throw another error at this [line](https://github.com/biocore/american-gut-web/blob/master/scripts/ag#L301)

The easy fix is to force the connection to log into the default postsql database.

But maybe I'm missing something. @beaugunderson - I remember you setup the installation doc.  Did you run into this problem before?

**Comment by beaugunderson** [comment]
Ah, my understanding was that psycopg2, in the absence of the `database` parameter, would connect to the default database. If that's not the case then I agree that the fix is to connect to explicitly connect to the default database. :)

The installation doc is based on my trial and error attempts to get everything set up locally and so may not be entirely complete; it's also from an OS X perspective.

**Comment by mortonjt** [comment]
No worries.  The installation guide is actually pretty good!  I only had to findle with the PATH variable a little bit to find pg_config and create a user and specify their permissions.

I got the fix from this [stackoverflow post](http://stackoverflow.com/questions/19426448/creating-a-postgresql-db-using-psycopg2)

Here's actual [documentation](http://postgresapp.com/documentation/gui-tools.html) to support that
